app/src/org/commcare/views/widgets/AbstractUniversalDateWidget.java

```diff
@@ -436,7 +450,7 @@ private void updateDateDisplay(long millisFromJavaEpoch) {
     /**
      * Update the widget helper date text (useful for those who don't know the other calendar)
      */
-    private void updateGregorianDateHelperDisplay() {
+    protected void updateGregorianDateHelperDisplay() {
```

Comment:
Ah, I think I misunderstood the use of the word `Helper`. I thought it was a helper method, but it is a method that shows helper text to the user. Ignore my initial suggestion.